python_src/src/lamp_py/performance_manager/flat_file.py

```diff
@@ -0,0 +1,130 @@
+import sqlalchemy as sa
+import pyarrow
+
+from lamp_py.postgres.postgres_schema import (
+    VehicleEvents,
+    VehicleTrips,
+    StaticStopTimes,
+)
+from lamp_py.postgres.postgres_utils import DatabaseManager
+
+
+def generate_daily_table(
+    db_manager: DatabaseManager, service_date: int
+) -> pyarrow.Table:
+    """
+    Generate a dataframe of all events and metrics for a single service date
+    """
+    query = (
+        sa.select(
+            VehicleEvents.stop_sequence,
+            VehicleEvents.stop_id,
+            VehicleEvents.parent_station,
+            VehicleEvents.vp_move_timestamp.label("move_timestamp"),
+            sa.func.coalesce(
+                VehicleEvents.vp_stop_timestamp,
+                VehicleEvents.tu_stop_timestamp,
+            ).label("stop_timestamp"),
+            VehicleEvents.travel_time_seconds,
+            VehicleEvents.dwell_time_seconds,
+            VehicleEvents.headway_trunk_seconds,
+            VehicleEvents.headway_branch_seconds,
+            VehicleTrips.service_date,
+            VehicleTrips.route_id,
+            VehicleTrips.direction_id,
+            VehicleTrips.start_time,
+            VehicleTrips.vehicle_id,
+            VehicleTrips.branch_route_id,
+            VehicleTrips.trunk_route_id,
+            VehicleTrips.stop_count,
+            VehicleTrips.trip_id,
+            VehicleTrips.vehicle_label,
+            VehicleTrips.vehicle_consist,
+            VehicleTrips.direction,
+            VehicleTrips.direction_destination,
+            VehicleTrips.static_start_time,
+            StaticStopTimes.arrival_time.label("scheduled_arrival_time"),
+            StaticStopTimes.departure_time.label("scheduled_departure_time"),
```

Comment:
We didn't give them these two columns previously, could be dropped.

Reply:
i feel like these are nice for them to have? i'll ask in slack.